media-fonts/inconsolata-lgc/inconsolata-lgc-1.3.0.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
Since we have EAPI=7 for font now, could you bump to newer EAPI?

Reply:
I haven't yet the slightest idea to make it work. Dropping `inherit font` helps, but it fails. 

https://dev.gentoo.org/~mgorny/articles/the-ultimate-guide-to-eapi-7.html

doesn't even bother to mention the word font let alone anything having changed with font.eclass. That's the only thorough doc I can find on the topic of moving to EAPI 7.

The emerge install dies with

```
make: *** No rule to make target 'install'.  Stop.
```

There are nearly no ebuilds in `media-fonts` that are tagged as EAPI 7 and referencing them didn't aide me. Do you have suggestions or perhaps information I can use to craft the ebuild properly for EAPI 7?

Reply:
Eh? I cannot reproduce that. I changed to EAPI=7 and your ebuild just works fine?
Note: font.eclass just got a few days back the EAPI=7 stuff. Is your local portage in sync?

Reply:
Just did a sync, still having a problem. If I leave `inherit font` then I get this attempting to digest the ebuild again:

```
EAPI 7 is not supported by font.eclass.
```

Reply:
That doesn't sound like your local portage is in sync. Please re-check. Maybe your used mirror is out of date?

Reply:
Verified the new contents of `/usr/portage/eclass/font.eclass` and re-tested with EAPI 7. Everything is functioning. Syncing up with a different mirror was indeed the solution. Thank you!